Data/Sentinel1.qmd

```diff
@@ -27,3 +27,108 @@ Sentinel data products are made available systematically and free of charge to a
+|------------ | ---------------------- | ---------------------- | ------------ | ------------ | ------------| -----------|
+|GRD | SAFE with Cloud Optimized GeoTIFF (COG_SAFE) (packed or unpacked) | IAD (Immediately available data) | World | Oct-14 - Present| [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1){target="_blank"}| Jul-23 (\* ** \*)|
+|GRD | original SAFE (packed) | IAD | World | Last 1 year| [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1){target="_blank"}| Jul-23 (\* *** \*)|
+|GRD | original SAFE (packed) | DAD (Deferred available data) | World | Oct-14 - Present-1 year| [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1){target="_blank"}| Jul-23|
+|OCN | packed or unpacked | IAD | World | Dec-14 - Present | [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-2){target="_blank"}| Jan-23|
+|SLC | packed or unpacked | IAD | World | Oct-14 - Present | [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-1){target="_blank"}| Jul-23|
+|L0 (RAW) | packed or unpacked | IAD | Europe | Oct-14 - Present | [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-0){target="_blank"}| Jul-23|
+|L0 (RAW) | packed or unpacked | IAD | RoW | Last 1 year | [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-0){target="_blank"}| Jul-23|
+|L0 (RAW) | packed | DAD | World | Oct-14 - Present-1 year | [Details](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/product-types-processing-levels/level-0){target="_blank"}| Jan-23(*)|
+
+(\***\*) Conversion of Sentinel-1 GRD products to the SAFE with Cloud Optimized GeoTIFF (COG_SAFE) format will begin in May 2023. The newest products will be converted and available first, and older products will be added gradually until the entire archive is converted. More details about how the original Sentinel-1-GRD products were converted to COG_SAFE format are [available below](@convert-cogsafe) .
+
+(\****\*) Original Sentinel-1 GRD products older than one year will be accessible with deferred availability (DAD), while the entire archive of COG_SAFE products will be available immediately (IAD). In case original Sentinel-1 GRD products would be needed with immediate access, users can convert COG_SAFE products to the original SAFE products using [this tool](https://github.com/eu-cdse/utilities/blob/main/COG2GRD.sh). 
+
+## [How to search for COG_SAFE products with OData API?]{#search-cogsafe-odata}
```

Comment:
Maybe API related information can be included somewhere in API section? Just a thought.